python/test/unit/test_contexts.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# test_contexts.py
+
+import unittest
+
+import pyarb as arb
+
+# to be able to run .py file from child directory
+import sys, os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../')))
+
+try:
+    import options
+except ModuleNotFoundError:
+    from test import options 
+
+"""
```

Comment:
Again, I would just have the single string: "all tests for non-distributed arb.context"

Reply:
Done.